lib/rtcpeerconnection/edge.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+var EventTarget = require('../util/eventtarget');
+var inherits = require('util').inherits;
+var util = require('../util');
+
+var match = navigator.userAgent.match(/Edge\/(\d+).(\d+)$/);
```

Comment:
Can you do something like

```js
var match = typeof navigator !== 'undefined'
  ? navigator.userAgent.match(/Edge\/(\d+).(\d+)$/)
  : null;
```

so that it's safe to require this in an environment where `navigator` is `undefined` (like Node)?